src/i18n/fr/ui.ts

```diff
@@ -18,10 +18,13 @@ export default UIDictionary({
 	'rightSidebar.a11yTitle': 'Table des matières',
 	'rightSidebar.onThisPage': 'Sur cette page',
 	'rightSidebar.overview': 'Vue générale',
-	'rightSidebar.more': 'Plus',
+	'rightSidebar.community': 'Communauté',
+	'rightSidebar.joinDiscord': 'Rejoignez nous sur Discord',
```

Comment:
"Rejoindre notre discord" could fit more on pair with the other links I think

Reply:
the imperative in this case sound more like an order to the reader, which is not what we want (i think 🤣)

Reply:
🤣 Haha, OK! Wasn’t sure because in English you can use the imperative “Join us on Discord” like this. OK, so:

```suggestion
	'rightSidebar.joinDiscord': 'Rejoindre nous sur Discord',
```

Reply:
"Rejoindre nous sur Discord" is not french anymore lol 🤣, more like that
```suggestion
	'rightSidebar.joinDiscord': 'Rejoindre notre Discord',
```

Reply:
Oops, thankfully _one_ of us is awake 😅